> Thanks for the contribution! Looks like the CI for the PP test for this model is failing. I think you need to use `get_pp_group()` inside the `forward` method of the model as well.

I check the ci log, the log:

```
[2024-09-22T14:01:53Z] Traceback (most recent call last):
--
  | [2024-09-22T14:01:53Z]   File "/usr/local/lib/python3.12/dist-packages/transformers/models/auto/configuration_auto.py", line 993, in from_pretrained
  | [2024-09-22T14:01:53Z]     config_class = CONFIG_MAPPING[config_dict["model_type"]]
  | [2024-09-22T14:01:53Z]                    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^
  | [2024-09-22T14:01:53Z]   File "/usr/local/lib/python3.12/dist-packages/transformers/models/auto/configuration_auto.py", line 695, in __getitem__
  | [2024-09-22T14:01:53Z]     raise KeyError(key)
  | [2024-09-22T14:01:53Z] KeyError: 'qwen2_vl'

```

qwen2 vl need the latest transformer library, how can we fix that.

--- a/tests/distributed/test_pipeline_parallel.py
+++ b/tests/distributed/test_pipeline_parallel.py
@@ -8,6 +8,8 @@ WARNING: This test runs in both single-node (4 GPUs) and multi-node
 import os
 
 import pytest
+from packaging import version
+from transformers import __version__ as transformers_version
 
 from vllm.logger import init_logger
 
@@ -46,6 +48,11 @@ def test_compare_tp(TP_SIZE, PP_SIZE, EAGER_MODE, CHUNKED_PREFILL,
     if VLLM_MULTI_NODE and DIST_BACKEND == "mp":
         pytest.skip("Skipping multi-node pipeline parallel test for "
                     "multiprocessing distributed backend")
+
+    # Skip tests that require transformers>=4.45.0
+    if "Qwen2-VL" in MODEL_NAME and version.parse(
+            transformers_version) < version.parse("4.45.0.dev0"):
+        pytest.skip("This test requires transformers>=4.45.0")
 
     pp_args = [
         # use half precision for speed and memory savings in CI environment

